extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -739,12 +739,13 @@ export class Bind {
 
       case 'class':
         initialValue = [];
-        // Ignore internal AMP classes.
         for (let i = 0; i < element.classList.length; i++) {
           const cssClass = element.classList[i];
+          // Ignore internal AMP classes.
           if (AMP_CSS_RE.test(cssClass)) {
-            initialValue.push(cssClass);
+            continue;
```

Comment:
Nit: Save a line by flipping the condition and putting the push inside the if-statement. (The comment makes the inverted condition easily readable)